nf_core/pipelines/params_file.py

```diff
@@ -213,7 +213,7 @@ def format_param(
             out += _print_wrapped("Required", mode="none", indent=4)
 
         out += _print_wrapped("\n", mode="end")
-        out += f"# {name} = {json.dumps(default)}\n"
+        out += f"# {name}: {json.dumps(default)}\n"
```

Comment:
```suggestion
        out += f"# '{name}': '{json.dumps(default)}'\n"
```
if I am not wrong this should handle names and entries with hyphens, which would break the syntax otherwise.

Reply:
Hi @mashehu thanks for the suggestion!

However, I am not able to reproduce this issue with the hyphens. 
I have added a parameter with a hyphen, `test-parameter`, to the config file, and it is correctly written to yaml output: 

```
## -----------------------------------------------------------------------------
## test-parameter
## -----------------------------------------------------------------------------
##     Dummy parameter to test the hyphen parsing
##     Type: string
## -----------------------------------------------------------------------------
# test-parameter: null
```

Am I missing something?

Reply:
okay, nevermind then, maybe that's only the case for json files then. This PR is good to go then.